src/permissions/compute.ts

```diff
@@ -10,7 +10,8 @@ import {
 import { getKeys } from './util';
 
 interface Permissions {
-  [name: string]: boolean;
+  [name: string]: boolean | unknown;
+  $errors: unknown;
```

Comment:
You need to type this differently when using it later. Why don't we add the correct typing here?

```suggestion
  $errors: Record<string, unknown>;
```

Reply:
Took type suggestion from Rober, should be addressed.